src/repl/command.rs

```diff
@@ -342,10 +381,10 @@ impl CommandInput {
     }
 
     fn usage(&self) -> String {
-        if self.hidden_reader.is_some() {
-            format!("{} (enter in hidden input)", self.usage)
-        } else {
-            self.usage.to_owned()
+        match self.type_ {
+            InputType::Optional => format!("[optional] {}", self.usage),
```

Comment:
Don't square brackets by themselves conventionally denote that the argument is optional?

Reply:
suppose we could use that convention anyhow!